packages/navigator/src/browser/navigator-tree.ts

```diff
@@ -66,13 +66,13 @@ export namespace WorkspaceNode {
         return CompositeTreeNode.is(node) && node.name === WorkspaceNode.name;
```

Comment:
Should not be here `node.id === WorkspaceNode.id`, since we can have different `name` for `WorkspaceNode` now?